content/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -28,12 +28,18 @@ perfect solution, but here are some ideas:
 - Install an actual Linux distro (e.g. Ubuntu) on a virtual machine, or in a HD
   partition (dual boot).
 
-## Disable short-file name generation
+## Enable symbolic links
 
-With NTFS, users may want to disable `8dot3` as per
-[this article](https://support.microsoft.com/en-us/help/121007/how-to-disable-8-3-file-name-creation-on-ntfs-partitions)
-to disable the short-file name generation. It is important to do so for better
-performance when the user has over 300K files in a single directory.
+> This is done automatically by the DVC's Windows
```

Comment:
This is done automatically if you installed DVC from the installer package ... You would probably need this if you install it via other methods- pip, conda, etc

(be more explicit)

Reply:
OK. Updated.